sdk/eventhub/Azure.Messaging.EventHubs.CheckpointStore.Blob/src/BlobPartitionManager.cs

```diff
@@ -49,17 +49,19 @@ public sealed class BlobPartitionManager : PartitionManager
         ///   Retrieves a complete ownership list from the storage blob service.
         /// </summary>
         ///
+        /// <param name="fullyQualifiedNamespace">The fully qualified host name for the Event Hubs namespace the ownership are associated with.</param>
         /// <param name="eventHubName">The name of the specific Event Hub the ownership are associated with, relative to the Event Hubs namespace that contains it.</param>
         /// <param name="consumerGroup">The name of the consumer group the ownership are associated with.</param>
         ///
         /// <returns>An enumerable containing all the existing ownership for the associated Event Hub and consumer group.</returns>
         ///
-        public override async Task<IEnumerable<PartitionOwnership>> ListOwnershipAsync(string eventHubName,
+        public override async Task<IEnumerable<PartitionOwnership>> ListOwnershipAsync(string fullyQualifiedNamespace,
```

Comment:
It might be helpful for the user if we provide an example of `fullyQualifiedNamespace` in the docs. Something like:
```
<param name="fullyQualifiedNamespace">The fully qualified Event Hubs namespace. This is likely to be similar to
<yournamespace>.servicebus.windows.net</param>
```